src/System.Data.SqlClient/tests/Tools/TDS/TDS/LanguageString.cs

```diff
@@ -0,0 +1,346 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.SqlServer.TDS
+{
+    /// <summary>
+    /// Class that contains langauge names and can convert between enumeration and strings
+    /// </summary>
+    public static class LanguageString
+    {
+        private const string English = "us_english";
+        private const string German = "Deutsch";
+        private const string French = "Fran\u00E7ais";
+        private const string Japanese = "\u65E5\u672C\u8A9E";
+        private const string Danish = "Dansk";
+        private const string Spanish = "Espa\u00F1o";
+        private const string Italian = "Italiano";
+        private const string Dutch = "Nederlands";
+        private const string Norwegian = "Norsk";
+        private const string Portuguese = "Portugu\u00EAs";
+        private const string Finnish = "Suomi";
+        private const string Swedish = "Svenska";
+        private const string Czech = "\u010De\u0161tina";
+        private const string Hungarian = "magyar";
+        private const string Polish = "polski";
+        private const string Romanian = "rom\u00E2n\u0103";
+        private const string Croatian = "hrvatski";
+        private const string Slovak = "sloven\u010Dina";
+        private const string Slovenian = "slovenski";
+        private const string Greek = "\u03B5\u03BB\u03BB\u03B7\u03BD\u03B9\u03BA\u03AC";
+        private const string Bulgarian = "\u0431\u044A\u043B\u0433\u0430\u0440\u0441\u043A\u0438";
+        private const string Russian = "\u0440\u0443\u0441\u0441\u043A\u0438\u0439";
+        private const string Turkish = "T\u00FCrk\u00E7e";
+        private const string BritishEnglish = "British";
+        private const string Estonian = "eesti";
+        private const string Latvian = "latvie\u0161u";
+        private const string Lithuanian = "lietuvi\u0173";
+        private const string Brazilian = "Portugu\u00EAs (Brasil)";
+        private const string TraditionalChinese = "\u7E41\u9AD4\u4E2D\u6587";
+        private const string Korean = "\uD55C\uAD6D\uC5B4";
+        private const string SimplifiedChinese = "\u7B80\u4F53\u4E2D\u6587";
+        private const string Arabic = "Arabic";
+        private const string Thai = "\u0E44\u0E17\u0E22";
+        private const string Bokmal = "norsk (bokm\u00E5l)";
+
+        /// <summary>
+        /// Convert a language to enumeration
+        /// </summary>
+        public static LanguageType ToEnum(string value)
+        {
+            // Check every langauge
+            if (string.Compare(value, English, true) == 0)
```

Comment:
Simply because ```string.Compare(...) == 0``` is a longer-version of ```string.Equals(...)``` and we generally try to use the latter instead of the former.  It's not a big deal, though.